### ISSUE TITLE
PyTorch 1.3.0 multinomial interface change

### ISSUE THREAD

**Issue by fKunstner** [f-dangel/backpack#7]
[PyTorch 1.3.0](https://github.com/pytorch/pytorch/releases/tag/v1.3.0) [changed the acceptable inputs](https://github.com/pytorch/pytorch/pull/25595) for the [multinomial function](https://pytorch.org/docs/stable/torch.html?highlight=multinomial#torch.multinomial).
The interface is now the same for the cuda and cpu versions.

This breaks the [Sampling of the symmetric factorization for the cross entropy loss](https://github.com/f-dangel/backpack/blob/master/backpack/core/derivatives/crossentropyloss.py#L30-L55).

Todo:
- [x] Add tests to sanity check the average of multiple runs of of `kfac` and `mc ggn` using low-precision `torch.allclose`.
- [x] Make those tests optional (they will take a while to run) and document how to run them.
- [x] Change the call to `multinomial` in [`derivatives/crossentropyloss.py`](https://github.com/f-dangel/backpack/blob/master/backpack/core/derivatives/crossentropyloss.py#L30-L55) to reflect the interface of PyTorch 1.3.0.
